app/models/validations/sales/household_validations.rb

```diff
@@ -20,6 +20,18 @@ def validate_previous_postcode(record)
     end
   end
 
+  def validate_buyers_living_in_property(record)
+    return unless record.form.start_date.year >= 2023
+
+    if record.buyers_will_live_in? &&
+        record.joint_purchase? &&
```

Comment:
isn't this redundant if we're also testing buyer one and buyer 2 after?

Reply:
Yes I think this is redundant given routing. good spot.

Reply:
I think that also makes this condition short enough to go on one line which is an added bonus